woodwork/table_accessor.py

```diff
@@ -206,7 +206,6 @@ def __getitem__(self, key):
 
         series = self._dataframe[key]
         column = copy.deepcopy(self._schema.columns[key])
-        column.semantic_tags -= {'index', 'time_index'}
         if column.use_standard_tags:
             column.semantic_tags |= column.logical_type.standard_tags
```

Comment:
Do we want to remove standard tags here too?

Reply:
yes! It should have the same behavior as the indexers

Reply:
✅  5fe0b07